scripts/prepare_rust_deps.py

```
@@ -46,7 +56,7 @@ def prepare_crate_as_vendored_dep(crate_path: Path):
     checksums = generate_cargo_checksum(crate_path)
     with tarfile.open(crate_path) as tarball:
         folder_name = tarball.getnames()[0].split("/")[0]
-        tarball.extractall(crate_path.parent, filter="data")
+        tarball.extractall(crate_path.parent, filter=_crate_tar_filter)
```

Ruff (0.16.1)

[error] Uses of `tarfile.extractall()`

(S202)